apps/dashboard/src/pages/index.tsx

```diff
@@ -16,14 +16,15 @@ function Home() {
   }, [])
 
   return (
+    
     <div className="main">
       <h1 className="text-3xl font-extrabold underline">Home</h1>
       <div className="nav">
-        <ul>
+        {/* <ul>
           <li>
             <Link href="/login">Login</Link>
           </li>
```

Comment:
I think it's good to avoid commented out code. If you don't think it's needed, best to remove it.